module/Database/src/Model/Member.php

```diff
@@ -594,8 +594,11 @@ public function setPaid(int $paid): void
      *
      * @return string|null
      */
-    public function getIban(): ?string
+    public function getIban($print = false): ?string
     {
+        if ($print) {
+            return preg_replace('/(\\w{4})/', '$1 ', $this->iban);
+        }
```

Comment:
Potentially passing `null` to `preg_replace()`, as `$this->iban` can be `null`.

Reply:
*this is deprecated

Reply:
[8fb8edb](https://github.com/GEWIS/gewisdb/pull/200/commits/8fb8edb178737bd2be7cd72116765a9b0f1725f2)